tests/customers/tests-customers.php

```diff
@@ -115,32 +115,6 @@ public function test_remove_payment_should_return_true() {
 		$this->assertFalse( in_array( self::$order, $order_ids ) );
 	}
 
-	public function test_increase_value_should_return_10() {
-		self::$customers[3]->increase_value( 10 );
-
-		$this->assertSame( 10.0, self::$customers[3]->purchase_value );
-	}
-
-	public function test_increase_purchase_count_should_return_1() {
-		self::$customers[3]->increase_purchase_count();
-
-		$this->assertSame( 1, self::$customers[3]->purchase_count );
-	}
-
-	public function test_decrease_value_should_return_90() {
-		self::$customers[4]->increase_value( 100 );
-		self::$customers[4]->decrease_value( 10 );
-
-		$this->assertSame( 90.0, self::$customers[4]->purchase_value );
-	}
-
-	public function test_decrease_purchase_count_should_return_0() {
-		self::$customers[3]->increase_purchase_count();
-		self::$customers[3]->decrease_purchase_count();
-
-		$this->assertSame( 0, self::$customers[1]->purchase_count );
-	}
-
```

Comment:
See above. Would be nice to keep these tests unless the methods are completely going away.